pkgs/os-specific/linux/ryzenadj/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, pciutils, cmake }:
+stdenv.mkDerivation rec {
+  pname = "ryzenadj";
+  version = "8.2.0";
+
+  src = fetchFromGitHub {
+    owner = "FlyGoat";
+    repo = "RyzenAdj";
+    rev = "b19903e9e487f32fa8e714fe9da0ff121030d657";
```

Comment:
When fetching a commit instead of a release tag, you should be using the `unstable-<commit-date>` for the `version` attribute.

Reply:
Replaced rev with correct version reference.